src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateLayoutModels()
             {
                 _rowsModel.CellChildMap[i, 0] = i;
                 _columnsModel.CellChildMap[0, i] = i;
-                _rowsModel.RowPercents[i] = _multiplier / ZoneCount; // _columnsModel is sharing the same array
+                _rowsModel.RowPercents[i] = ((_multiplier * (i + 1)) / ZoneCount) - ((_multiplier * i) / ZoneCount); // _columnsModel is sharing the same array

```

Comment:
Isn't this `_multiplier / ZoneCount` ?

Reply:
Same for other similar lines bellow.

Reply:
No, in integer arithmetic it's not, and that's kind of the whole point.

Reply:
Ivan, can you add comments in the code pointing out the fix is to round the zone size so it fills the screen? This is to avoid someone in the future to change the code ;)

Reply:
Sure!